scripts/get-examples.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I thought of making that change but it requires updating the README in this and all dependant repositories and so I decided to skip for now.